client/components/sub-masterbar-nav/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { find, noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Navbar from './navbar';
+import Dropdown from './dropdown';
+import Item from './item';
+
+const OptionShape = PropTypes.shape( {
+	label: PropTypes.string.isRequired,
+	uri: PropTypes.string.isRequired,
+	icon: PropTypes.string
+} );
+
+export default class SubMasterbarNav extends Component {
+	static propTypes = {
+		fallback: OptionShape,
+		options: PropTypes.arrayOf( OptionShape ),
+		uri: PropTypes.string.isRequired
+	};
+
+	static defaultProps = {
+		options: []
+	}
+
+	render() {
+		return (
+			<div className="sub-masterbar-nav">
+				<Dropdown selected={ this.getSelected() }>
+					{ ( { onSelect } ) => (
+						this.renderItems( onSelect )
+					) }
+				</Dropdown>
+				<Navbar>
+					{ this.renderItems( noop ) }
```

Comment:
The previous comment about children callback makes even more sense here. Why have general renderItems if in just second usage you need to pass `noop` when there should be just nothing in more specific render inside `Navbar`